src/adaptors/notional-v3/index.js

```diff
@@ -0,0 +1,197 @@
+const { request, gql } = require('graphql-request');
+const utils = require('../utils');
+const superagent = require('superagent');
+const { default: BigNumber } = require('bignumber.js');
+
+const API = (chain) => `https://data-dev.notional.finance/${chain}/yields`
+const NOTE_Mainnet = '0xCFEAead4947f0705A14ec42aC3D44129E1Ef3eD5'
+
+const SUBGRAPHS = {
+  arbitrum: 'https://api.studio.thegraph.com/query/33671/notional-finance-v3-arbitrum/version/latest'
```

Comment:
FYI: this is a development URL in studio, are you guys able to support queries against the decentralized subgraph network or do you need to use the hosted network only?